docs/getting-started.md

```
@@ -120,14 +120,14 @@ Cloudflare resources or rewrite `wrangler.jsonc` resource IDs.
 
 See `docs/agents/setup.md` for local dev commands and verification steps.
 
-If you want realistic local sample data, run:
+To create a deterministic local test login, run:
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)